solidity/package.json

```diff
@@ -28,7 +29,7 @@
     "@keep-network/tbtc": ">1.1.2-dev <1.1.2-ropsten",
     "@openzeppelin/contracts": "^4.1.0",
     "@tenderly/hardhat-tenderly": "^1.0.12",
-    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#507c647"
+    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#4985bcf"
```

Comment:
Non-blocking but it should go in a separate commit. "Test cleanup" is not a good commit message for such an important version bump :)